pom.xml

```diff
@@ -67,9 +67,9 @@
         <jenkins.version>2.121.1</jenkins.version>
         <java.level>8</java.level>
         <workflow-step-api-plugin.version>2.15</workflow-step-api-plugin.version>
-        <workflow-cps-plugin.version>2.32</workflow-cps-plugin.version>
-        <workflow-support-plugin.version>2.14</workflow-support-plugin.version>
-        <workflow-api-plugin.version>2.28</workflow-api-plugin.version>
+        <workflow-cps-plugin.version>2.58-beta-1</workflow-cps-plugin.version>
```

Comment:
These can be updated to non-beta versions now.

Reply:
Remind me to do so once https://github.com/jenkinsci/workflow-durable-task-step-plugin/pull/75 is in a release.

Reply:
Ok, if that PR isn't released by next Friday I will release it and ping you then.